packages/js/product-editor/assets/icons/section_images.svg

```diff
@@ -0,0 +1,20 @@
+<svg
+	width="24"
+	height="25"
+	viewBox="0 0 24 25"
```

Comment:
```suggestion
	view-box="0 0 24 24"
```

You have a typo here.
Also I imagine this kind of icon should be perfectly squared. (24x24)